app/src/components/account/UserMenu.vue

```diff
@@ -166,7 +166,9 @@ export default {
           )
           .map((address) => ({
             ...address,
-            sessionType: address.type || "explore",
+            sessionType: address.sessionType
```

Comment:
for which case did this not work? is this now ever "explore"?

Reply:
no, it was ever "Explore" before. The problem is that it wasn't considering when `address` already has `addressType` as a field